### PR TITLE
AIRFLOW-91: ssl gunicorn support

### PR DESCRIPTION
**Pull request by zstan**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- AIRFLOW-91:

### CODE REVIEW

File: airflow/configuration.py [hunk 1]
```diff
@@ -109,6 +109,8 @@ def run_command(command):
         'expose_config': False,
         'workers': 4,
         'worker_class': 'sync',
+        'ssl_certfile': None,
```
Comment (mistercrunch) [inline airflow/configuration.py:112]: Is this used anywhere? I don't think it's bound to anything.

File: airflow/configuration.py [hunk 2]
```diff
@@ -263,6 +265,12 @@ def run_command(command):
 # Filter the list of dags by owner name (requires authentication to be enabled)
 filter_by_owner = False
 
+# SSL certificate file
+# ssl_certfile =
```
Comment (mistercrunch) [inline airflow/configuration.py:269]: Not bound.

**Comment by bolkedebruin** [comment]
@zstan @mistercrunch the commit was reverted. @zstan please make sure you we can use the config variables and update it in a new PR. Thanks.

**Comment by r39132** [comment]
This was already fixed by #1760.

**Pull request merged**